tests/integration_tests/datasources/test_oci_networking.py

```diff
@@ -0,0 +1,158 @@
+import re
+from typing import Iterator, Set
+
+import oci
+import pytest
+import yaml
+from pycloudlib.oci.utils import wait_till_ready
+
+from tests.integration_tests.clouds import IntegrationCloud
+from tests.integration_tests.instances import IntegrationInstance
+from tests.integration_tests.util import verify_clean_log
+
+DS_CFG = """\
+#cloud-config
+datasource:
+  Oracle:
+    configure_secondary_nics: {configure_secondary_nics}
+"""
+
+
+def customize_environment(
+    client: IntegrationInstance,
+    tmpdir,
+    configure_secondary_nics: bool = False,
+):
+    assert client.execute("rm -f /run/initramfs/open-iscsi.interface").ok
```

Comment:
My fault because I suggested it, but these next two `execute`s don't do anything meaningful. On reboot,  `/run/` gets recreated.

`/etc/cloud/cloud.cfg.d/99-disable-network-config.cfg"` is a bit of a misnomer on Oracle. It just means network can't be configured from the /etc/cloud directory. Removing it isn't necessary.

Reply:
Removed.